packages/atlas/src/views/viewer/EditMembershipView/EditMembershipView.tsx

```diff
@@ -78,17 +126,25 @@ export const EditMembershipView: FC = () => {
     memberId ? [memberId, watch('handle'), createMemberInputMetadata(watch())] : undefined
   )
 
-  const handleEditMember = handleSubmit(async (formData) => {
+  const handleEditMember = handleSubmit(async (data) => {
     if (!joystream || !activeMembership) {
       return
     }
 
     const success = await handleTransaction({
       txFactory: async (updateStatus) => {
-        const memberInputMetadata: MemberInputMetadata = createMemberInputMetadata(formData)
+        let fileUrl = ''
+        if (data.avatar?.blob && dirtyFields.avatar) {
+          fileUrl = await uploadAvatarImage(data.avatar.blob)
```

Comment:
I think we should handle failure a bit better here. Currently if the avatar service is unavailable, it will look like the transaction failed and nothing in the console points to the avatar service being the issue:
<img width="1568" alt="CleanShot 2022-08-25 at 15 45 42@2x" src="https://user-images.githubusercontent.com/12646744/186681708-74bdba8e-a300-4146-b5de-0faff93252f7.png">

I think ideally we would show some snackbar mentioning that it's the avatar that couldn't be uploaded so that users know that they can get rid of avatar and proceed anyway. The same applies to new member creation